autoload/test/javascript/webdriver.vim

```diff
@@ -0,0 +1,30 @@
+if !exists('g:test#javascript#webdriver#file_pattern')
+  let g:test#javascript#webdriver#file_pattern = '\vtest/specs/.*\.js$'
+endif
+
+function! test#javascript#webdriver#test_file(file) abort
+  return a:file =~# g:test#javascript#webdriver#file_pattern
+    \ && !empty(glob('wdio.conf.js'))
+endfunction
+
+function! test#javascript#webdriver#build_position(type, position) abort
+  return []
+
+  if a:type == 'file'
+    return [a:position['file']]
```

Comment:
According to https://github.com/janko-m/vim-test/issues/225#issue-261966924 this should be 

```vim
return ["--spec", a:position['file']]
```